.template-lintrc.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+  extends: 'octane',
+
+  // TODO: Turn on a rule and fix errors
+  rules: {
```

Comment:
These are exceptions that I had to make after installing the latest `ember-template-lint`. We'll open issues to Octanify the app and remove these exceptions.